From 55b09e4855bc4298ffce38ca6e81b2893cdaa93f Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Fri, 16 Jan 2015 21:11:37 -0800
Subject: [PATCH 3002/3005] buildit: Set compatibility version to
 RC_ProjectSourceVersion

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit ce37e96b968fbcc4b6aef18b1a163d297db903b2)
---
 lib/buildit | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git llvm_release_34/projects/libcxx/lib/buildit macports_release_34/projects/libcxx/lib/buildit
index cdb7826..1549974 100755
--- llvm_release_34/projects/libcxx/lib/buildit
+++ macports_release_34/projects/libcxx/lib/buildit
@@ -49,7 +49,8 @@ case $TRIPLE in
     then
         EXTRA_FLAGS="-nostdinc++ -std=c++11 -U__STRICT_ANSI__"
         LDSHARED_FLAGS="-o libc++.1.dylib \
-            -dynamiclib -nodefaultlibs -current_version 1 \
+            -dynamiclib -nodefaultlibs \
+            -current_version ${RC_ProjectSourceVersion} \
             -compatibility_version 1 \
             -install_name /usr/lib/libc++.1.dylib \
             -Wl,-reexport_library,/usr/lib/libc++abi.dylib \
-- 
2.2.2

